DOCS/man/context_menu.rst

```diff
@@ -1,10 +1,49 @@
-CONTEXT MENU SCRIPT
-===================
-
-This script provides a context menu for platforms without integration with a
-native context menu. On these platforms, it can be disabled entirely using the
+CONTEXT MENU
+============
+
+The context menu is a menu that pops up on the video window. By default, it is
+bound to right click.
+
+menu.conf
+---------
+
+The default menu is defined in ``etc/menu.conf``. You can define your own menu
```

Comment:
Looks like `etc/menu.conf` is not installed, it's embedded in binary. So this reference in docs is not relevant.

Reply:
Removed since the actual URL is already shown